HeavyFlavorAnalysis/Onia2MuMu/src/OniaPhotonConversionProducer.cc

```diff
@@ -381,23 +380,25 @@ reco::Candidate::LorentzVector OniaPhotonConversionProducer::convertVector(const
 }
 
 
-void OniaPhotonConversionProducer::endJob(){
-   std::cout << "############################" << std::endl;
-   std::cout << "Conversion Candidate producer report" << std::endl;
-   std::cout << "############################" << std::endl;
-   std::cout << "Total examined conversions: " << total_conversions << std::endl;
-   std::cout << "Selection fail candidates:  " << selection_fail << std::endl;
-   std::cout << "Algo fail candidates:       " << algo_fail << std::endl;
-   std::cout << "Quality fail candidates:    " << flag_fail << std::endl;
-   std::cout << "Pi0 fail:                   " << pizero_fail << std::endl;
-   std::cout << "Total duplicates found:     " << duplicates << std::endl;
-   std::cout << "Vertex compatibility fail:  " << TkVtxC << std::endl;
-   std::cout << "Compatible inner hits fail: " << CInnerHits << std::endl;
-   std::cout << "Highpurity Subset fail:     " << highpurity_count << std::endl;
-   std::cout << "############################" << std::endl;
-   std::cout << "Final number of conversions:  " << final_conversion << std::endl;
-   std::cout << "Stored number of conversions: " << store_conversion << std::endl;
-   std::cout << "############################" << std::endl;
+void OniaPhotonConversionProducer::endStream() {
```

Comment:
so likely this is then not doing what @alberto-sanchez intends (which is to print the summary for the job)